File: src/01-easy/1d-bush-fire/solution.vb
Module Solution
    Sub Main()
        Dim n As Integer = CInt(Console.ReadLine())
        For i As Integer = 1 To n
            Dim s As String = Console.ReadLine().Trim()
            Dim d As Integer = 0, j As Integer = 0
            While j < s.Length
                If s(j) = "f"c Then d += 1 : j += 3 Else j += 1
            End While
            Console.WriteLine(d)
        Next
    End Sub
End Module

File: src/01-easy/1d-bush-fire/solution.cs
using System;
class Solution {
    static void Main(){
        int n=int.Parse(Console.ReadLine());
        while(n-->0){
            string s=Console.ReadLine().Trim(); int d=0,j=0;
            while(j<s.Length){if(s[j]=='f'){d++;j+=3;}else j++;}
            Console.WriteLine(d);
        }
    }
}
